bookings.json

```diff
@@ -48,5 +48,15 @@
     "roomId": 3,
     "checkInDate": "2017-08-30",
     "checkOutDate": "2017-10-02"
+  },
+  {
+    "id": 6,
+    "title": "Ms",
+    "firstName": "Farzaneh",
+    "surname": "Haghani",
+    "email": "farzaneh.haghani@gmail.com",
+    "roomId": 1,
+    "checkInDate": "2023-06-25",
```

Comment:
When using dates/times in software, always use ISO8601 format. In this case it might feel strange to include the hours/minutes/seconds as you're talking about specific days, however it's the standard convention for storing date/time information.